tests/data/staging/data_cleaning/test_run_live_or_frozen.csv

```diff
@@ -1,8 +1,8 @@
-target,error,live,frozen
-1,C,1,1
-2,E,2,
-3,O,3,3
-4,W,4,
-5,C,5,5
-6,E,6,
-7,W,7,
+target,error,live,frozen,frozen_error

```

Comment:
Why does frozen_error have a numerical type? Does this not relate to the other error in col2?

Reply:
Frozen_error contains the deleted adjusted values.